github_pr.py

```diff
@@ -42,6 +43,8 @@ def _print_pr(pr, **args):
     elif 'comments' in args and args['comments']:
         for comment in pr.get_comments():
             print "#%d : Comment - %s" % (pr.number, comment.body)
+    elif 'filters' in args and args['filters']:
+        print "%d" % (pr['pr'].number)
```

Comment:
If we want to choose what columns to output, maybe we should have a `--columns` flag that defaults to a reasonable set but you could set to `--columns number` and it would just output the number. I don't want to have the fact that someone chose to add filters change how it outputs, that doesn't make sense.